ajax/libs/package.json

```
@@ -0,0 +1,30 @@
+{
+  "name": "node-polyglot",
+  "filename": "index.js",
+  "description": "Give your JavaScript the ability to speak many languages.",
+  "author": "Spike Brehm (https://github.com/spikebrehm)",
+  "homepage": "https://github.com/airbnb/polyglot.js",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/airbnb/polyglot.js.git"
+  },
+  "license": "BSD-2-Clause",
+  "keywords": [
+    "i18n",
+    "internationalization",
+    "internationalisation",
+    "translation",
+    "interpolation",
+    "translate",
+    "polyglot"
+  ],
+  "npmName": "node-polyglot",
+  "npmFileMap": [
+    {
+      "basePath": "",
+      "files": [
+        "**/*"
```

```suggestion
        "index.js"
```
I believe you only require the single `index.js` file, so this should just be that. Currently, this glob will include every single file in the npm package, including docs as Zack said.